src/stores/events/EventsFilterStore.ts

```diff
@@ -66,6 +66,28 @@ export default class EventsFilterStore {
 		this.filter = filter;
 	}
 
+	@action
+	public raiseFilterTimestamp(mins: number) {
```

Comment:
There are linting errors
We can actually use one method for adding and subtracting minutes.
lets call this method `changeTimestamp` and we will pass -30 or 30 min. And let's use moment for this `moment.utc(filter.from).add(minutes, 'minutes')` would be a lot easier to read